### ISSUE TITLE
Edit Layers makes browser stop working

### ISSUE THREAD

**Issue by airtonjr** [geoman-io/leaflet-geoman#729]
Hello!

I am trying to use this plugin with `leaflet`  but when I am trying to click on the `Edit Layers` button my browser stop working. Includes `Chrome`, `Opera`, and `Firefox`.

Here is my code:

```
var m = L.map('map').locate({
        setView: true,       
    }), drawnItems = L.featureGroup().addTo(m);

 m.pm.addControls({
        position: 'topleft'
    });

m.on('pm:create', (e) => {
        var layer = e.layer;
        drawnItems.addLayer(layer);
    });
```

Any idea why would this happen? Or if some leaflet version is not supported?

Thank you in advance.

**Comment by Falke-Design** [comment]
I was not able to reproduce this. Can you please create a demo https://jsfiddle.net/o1dwu2vg/

What leaflet Version are you using? Do you use the latest version of geoman 2.8.0?

**Comment by gavinduffy2** [comment]
just an FYI, i have the same issue. i simply removed this tool from the toolbar. everything else works fantastic.
great plugin.

**Comment by Falke-Design** [comment]
@gavinduffy2 can you please give me more informations? How many layers are on the map? Only a certain Shape (Marker, Line, Polygon,...)? Are you using some other plugins?

The best would be to have a demo

**Comment by gavinduffy2** [comment]
Hi there

I tried not adding some layers to the map, but as soon as I click the edit icon, chrome dies regardless.
There is nothing to see in f12, the console fails to open, chrome beng dead

I have a basic OSM base layer and 1 or 2 geojson layers.
Sorry I can’t provide much more info.


[cid:image002.png@01D70EAC.C9458820]
G A V I N    D U F F Y
Office: 877-354-1170 ext. 111

**Comment by gavinduffy2** [comment]
Hi there

I took your sample, it worked fine as is.
I added the following code and a reference to jquery to load the attached geojson.
Then chrome hangs if I try to edit markup.
Confirm you get this as the script tags may spam the email.
If I don’t hear from you, I’ll resend without the tags.

<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>

    // load GeoJSON parcel polys
    var geojsonPolys;
        $.getJSON("maps/nj/parcels/parcel_1339.geojson", function (data) {
            geojsonPolys = L.geoJson(data, { style: styleParcel }).addTo(map);
        });


[cid:image002.png@01D70EB8.AE3291C0]
G A V I N    D U F F Y
Office: 877-354-1170 ext. 111

**Comment by Falke-Design** [comment]
@gavinduffy2 thx!!! It was possible to reproduce, I will take a look into it, in the next days.

```
<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>

// load GeoJSON parcel polys
var geojsonPolys;
$.getJSON("https://raw.githubusercontent.com/scdoshi/us-geojson/master/geojson/state/AK.geojson", function (data) {
	console.log(data);
  //geojsonPolys = L.geoJson(data).addTo(map);
});
```

**Comment by gavinduffy2** [comment]
No worries,  glad I could help.
Any thoughts on adding a text label control?

Gavin Duffy

________________________________

**Comment by Falke-Design** [comment]
There is a open Feature Request  #332 you can upvote it.
